.github/header-checker-lint.yml

```diff
@@ -18,6 +18,8 @@ ignoreFiles:
   - "**/ghcnd-stations.txt"
   - "texttospeech/snippets/resources/example.txt"
   - "texttospeech/snippets/resources/hello.txt"
+  - "language/**/*.txt"
+  - "language/v1/test/*.yaml"
```

Comment:
Will see if I can add these here. Could you perhaps try and see if you can add headers and if it would break the tests for now?

Reply:
Added specific paths.

Reply:
I guess I can add comments to YAMLs, as the syntax supports comments. However, I don't know about TXT files. The whole point of the product is parsing and understanding the samples - I'd have to rewrite the code to recognize comments, or just skip the copyright header somehow. I don't think that's in scope of this migration.